src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -392,19 +378,33 @@ main() {
   setup_workdir
   export NXF_WORK
 
+  # download default applet file type inputs
+  dx-download-all-inputs --parallel @@EXCLUDE_INPUT_DOWNLOAD@@ 2>/dev/null 1>&2
+  RUNTIME_CONFIG_CMD=''
+  RUNTIME_PARAMS_FILE=''
+  [[ -d "$HOME/in/nextflow_soft_confs/" ]] && RUNTIME_CONFIG_CMD=$(find "$HOME"/in/nextflow_soft_confs -name "*.config" -type f -printf "-c %p ")
+  [[ -d "$HOME/in/nextflow_params_file/" ]] && RUNTIME_PARAMS_FILE=$(find "$HOME"/in/nextflow_params_file -type f -printf "-params-file %p ")
+  if [[ -d "$HOME/in/docker_creds" ]]; then
+    CREDENTIALS=$(find "$HOME/in/docker_creds" -type f | head -1)
+    [[ -s $CREDENTIALS ]] && docker_registry_login || echo "no docker credential available"
+    dx upload "$CREDENTIALS" --path "$DX_WORKSPACE_ID:/dx_docker_creds" --brief --wait --no-progress || true
```

Comment:
make a copy of the creds file in job container so we can find it in subjob via its specific filename